lib/modules/platform/gitlab/index.ts

```diff
@@ -169,7 +170,7 @@ export async function getRepos(config?: AutodiscoverConfig): Promise<string[]> {
     });
     logger.debug(`Discovered ${res.body.length} project(s)`);
     return res.body
-      .filter((repo) => !repo.mirror)
+      .filter((repo) => config?.includeMirrors || !repo.mirror) // eslint-disable-line @typescript-eslint/prefer-nullish-coalescing
```

Comment:
don't suppress this lint error

Reply:
Without suppressing the lint error the build fails, which is why I used  
`const includeMirrors = config?.includeMirrors === true;` before, which @rarkins did not like.
What is the neatest way to fix this here?